core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -747,6 +747,7 @@ $(document).ready(function () {
                     "info": true,
                     "processing": true,
                     "lengthMenu": [[20, 40, 60, 100, totalTasksToShow], [20, 40, 60, 100, "All"]],
```

Comment:
Why not change `totalTasksToShow` to `-1` here?

Reply:
@sarutak  Great advice!  I am not familiar with `lengthMenu`. I just checked the reference https://datatables.net/reference/option/lengthMenu，set `totalTasksToShow ` to `-` is really the best solution

Reply:
@sarutak  done this and munual test works well.